### PR TITLE
Move CommonPaths to Backend

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Move CommonPaths to Shared to prepare for moving Storage and ListedItem.

**Details of Changes**
- This pr moves CommonPaths to Shared. It is necessary to move the storage to the backend soon.
- The CommonPaths file has been restyled and its dictionary has become readonly.
- A cleanup has been done in the using files that depend on CommonPaths.

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@cinqmilleans can you fix the build error?

**Comment by cinqmilleans** [comment]
@yair100 It compiles for me. I look at it. This appears to be a missing dependency.

File: src/Files.Backend/Files.Backend.csproj [hunk 1]
```diff
@@ -17,4 +17,14 @@
...
 
+  <ItemGroup>
+    <Reference Include="Windows.Foundation.FoundationContract">
+      <HintPath>..\..\..\..\Program Files (x86)\Windows Kits\10\References\10.0.22000.0\Windows.Foundation.FoundationContract\4.0.0.0\Windows.Foundation.FoundationContract.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+    </Reference>
+    <Reference Include="Windows.Foundation.UniversalApiContract">
+      <HintPath>..\..\..\..\Program Files (x86)\Windows Kits\10\References\10.0.22000.0\Windows.Foundation.UniversalApiContract\14.0.0.0\Windows.Foundation.UniversalApiContract.winmd</HintPath>
+    </Reference>
+  </ItemGroup>
```
Comment (lukeblevins) [inline src/Files.Backend/Files.Backend.csproj:28]: Why do we have a dotnet standard library referencing WinRT contracts?

**Comment by cinqmilleans** [comment]
This pr has a design problem and requires additional analysis. I close this pr.

**Pull request closed without merging**